### PR TITLE
Small fixes to make deployment better.

### PR DESCRIPTION
**Pull request by tanin47**
Fix here and there to make it easier to deploy to a different project.

### CODE REVIEW

File: ci/continuous_integration/daemon/main.py [hunk 1]
```diff
@@ -209,6 +209,10 @@ def clean():
   """Clean the repo and all of the sub repos. This combination seems to
     work. Please be careful when changing it."""
   process.call('git clean -ffdd', cwd=CHROMIUM_SRC)
+  # We've encountered the case where `git clean` and `git reset --hard` don't
```
Comment (tanin47) [inline ci/continuous_integration/daemon/main.py]: This part will likely be changed again. For now, it seems to work...

File: tool/clusterfuzz/resources/supported_job_types.yml [hunk 2]
```diff
@@ -8,6 +8,7 @@ presets: #Any parameters in a preset can be overridden in the actual definition
             reproducer: Base
         pdfium_chromium:
             preset: pdfium_standalone
+            source_name: chromium
```
Comment (tanin47) [inline tool/clusterfuzz/resources/supported_job_types.yml]: This is a bug.

Comment (inferno-chromium) [inline tool/clusterfuzz/resources/supported_job_types.yml]: Why ? Do you have a fix for this. I think this was there for building pdfium in chromium repo ?

Comment (tanin47) [inline tool/clusterfuzz/resources/supported_job_types.yml]: We have two of them. One is `pdfium_standalone` which builds from pdfium source. This is `pdfium_chromium` which builds from chromium source.

The bug is from the big refactoring a few days ago.

**Pull request merged**